src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/dto/json/OppdaterVilkårsvurderingJson.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.json
+
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvis
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Delvilkår
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vurdering
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.DelvilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.tilDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.hovedreglerPassBarn
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.RegelId
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.SvarId
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.vilkårsreglerPassBarn
+import java.util.UUID
+
+data class OppdaterVilkårsvurderingJson(
```

Comment:
Å ha med `Json` i navnet skurrer for meg. Det får meg å tenke på autogenererte object fra xsd som hadde suffix `Xml`. 
Det introduserer unødvendig redundans og binder navnet på Dto for tett til et spesifikt serialiseringsformat.

Reply:
Kan bytte til `RestDto` eller `ApiDto`, for eksempel? Intensjonen med navnet er å tydeliggjøre at dette objektet har én og bare én oppgave: Å eksponere data til konsumenter.

Reply:
VilkårDto eksponerer også kun data til konsumentene? 
Jeg ser egentlige ikke direkt noen forskjell på hvordan mapping/håndtering av VilkårDto og de objektene du har opprettet, begge delene mappes til Vilkår. På ulike måter, med ulik kompleksitet.